scenarios/traces/es-jaeger/Makefile

```diff
@@ -0,0 +1,47 @@
+NS="es-jaeger-ingest"
+
+all: namespace install
+
+.PHONY: helm-repo-es
+helm-repo-es:
+	-helm repo add elastic https://helm.elastic.co
+	helm repo update elastic
+
+.PHONY: helm-repo-jaeger
+helm-repo-jaeger:
+	-helm repo add jaegertracing https://jaegertracing.github.io/helm-charts
+	helm repo update jaegertracing
+
+.PHONY: helm-dep
+helm-dep:
+	-helm dep build helm/
+
+.PHONY: namespace
+namespace:
+	kubectl create namespace "$(NS)"
```

Comment:
Do we need a separate target when helm can do this for us with `--create-namespace`?

Reply:
Not anymore, I'll get that cleaned up.